docs/directives.md

```diff
@@ -0,0 +1,125 @@
+# Directives, or "magic commands"
+
+A directive is a scriptable shortcut to a more complex behavior. Directives are similar to the magic command concept familiar to IPython and Jupyter users. There is no distinction in .NET Interactive between a "cell magic" and a "line magic". 
```

Comment:
Is Directives the right noun? Considering how frequently it is used in C#.

Reply:
Changing to "magic commands".